packages/amazonq/.changes/next-release/Bug Fix-704d175a-a99f-4ce7-bf08-b94d6f7218c0.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Right-click context menu not sent if chat was closed"
```

Maybe we can make this more intuitive

```suggestion
	"description": "`Send to prompt` context menu not sent if Amazon Q chat was not already active"
```